go.mod

```diff
@@ -0,0 +1,37 @@
+module k8s.io/kube-openapi
+
+go 1.12
+
+require (
+	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
```

Comment:
were these primed from the godeps.json file? should we drop the Godeps dir or do we intend to maintain that file as well?

Reply:
> were these primed from the godeps.json file?

Yes

> should we drop the Godeps dir or do we intend to maintain that file as well?

My gut feeling would be that we should remove it, do you have a recommendation?

Reply:
I'd recommend removing it

Reply:
Done